covariance_estimation:
  enabled: false
  method: "RANGE" # Options: RANGE (default), RANGE_BIAS
  range_noise: 0.05
  range_bias_noise: 0.05
